### ISSUE TITLE
Ability to set export/backup dir

### ISSUE THREAD

**Issue by Friss** [dokku/dokku-redis#143]
Currently the directory for a backup uses `mktemp` which usually means it will end up in `/tmp`. 

I ran into an issue where I couldn't export my database because my `/tmp` directory is on a smaller disk while my DB runs on a much larger disk. 

So if I could set my own directory I can route the backups somewhere else that won't run out of room.

**Comment by Friss** [comment]
I actually just learned that you can pass `--tmpdir` to `mktemp` and it will automatically look for `$TMPDIR` otherwise default to `/tmp`.

**Comment by josegonzalez** [comment]
At least on a mac:

```
       -t     interpret TEMPLATE as a single file name component, relative to a directory: $TMPDIR, if set; else  the
              directory specified via -p; else /tmp [deprecated]
```

**Comment by Friss** [comment]
Ah I was looking at https://www.gnu.org/software/autogen/mktemp.html (and tested on a ubuntu machine which worked)

**Comment by josegonzalez** [comment]
Still says deprecated though.

**Comment by josegonzalez** [comment]
Implemented in 1.12.5 of this plugin.
